byron/chain/executable-spec/test/Cardano/Spec/Chain/STS/Properties.hs

```diff
@@ -72,3 +79,110 @@ signersListIsBoundedByK = property $ do
         signersListIsBoundedByKInState :: MonadTest m => BlockCount -> State CHAIN -> m ()
         signersListIsBoundedByKInState (BlockCount k') (_sLast, sgs, _h, _utxoSt, _ds, _us) =
           assert $ length sgs <= fromIntegral k'
+
+
+relevantCasesAreCovered :: Property
+relevantCasesAreCovered = withTests 400 $ property $ do
+  tr <- forAll $ traceSigGen (Maximum 250) (sigGenChain GenDelegation NoGenUTxO)
+  let certs = traceDCerts tr
+
+  -- for at least 15% of traces...
+  cover 15
+        "there are more certificates than blocks"
+        (traceLength tr <= length certs)
+
+  -- for at least 5% of traces...
+  cover 5
+        "the majority of blocks (at least 75%) have certificates"
+        (emptyDelegationPayloadRatio (traceDCertsByBlock tr) <= 0.25)
+
+  -- for at least 50% of traces...
+  cover 50
```

Comment:
This is really weird, I would expect a higher percentage of delegations to this epoch. I'm thinking we might want to make an issue to investigate this ...

Reply:
will do